Tests/UnitTests/Scripts/tests.ts

```diff
@@ -567,7 +583,7 @@ describe("URL", function () {
     it("should update href after URLSearchParams are changed", function () {
         // Augment URL with pathname and search
         const url = new URL("https://httpbin.org/en-US/docs?foo=1&bar=2");
-        url.searchParams.set("foo", 999);
+        url.searchParams.set("foo", 999 as any);
```

Comment:
Why this cast? Should it just be
```suggestion
        url.searchParams.set("foo", "999");
```